apps/mobile-wallet/src/screens/new-wallet/AddBiometricsScreen.tsx

```diff
@@ -53,16 +53,16 @@ const AddBiometricsScreen = ({ navigation, ...props }: AddBiometricsScreenProps)
   const skipAddressDiscovery = method === 'create' || addressIds.length > 1
 
   const activateBiometrics = () => {
-    dispatch(biometricsToggled())
+    dispatch(allBiometricsEnabled())
```

Comment:
Nit: I find `allBiometricsEnabled` a bit confusing. Is there "multiple biometrics"? Couldn't we just say `biometricsEnabled`? :)

Reply:
There is:
- `biometricsRequiredForAppAccess`
- `biometricsRequiredForTransactions`